sirepo/api_auth.py

```diff
@@ -6,7 +6,7 @@
 """
 from __future__ import absolute_import, division, print_function
 from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
-from pykern import pkcollections
+from pykern import pkcollections, pkconfig
```

Comment:
make it's own import. once we have a formatter, this won't be an issue.

Reply:
b7a43c1b48e8e47121aa5d2263a14432e5409dcd